app/src/main/java/com/onboarding/calculator/mvp/view/ActivityView.java

```diff
@@ -0,0 +1,18 @@
+package com.onboarding.calculator.mvp.view;
```

Comment:
this class can be in a package called **base** inside the view package

Reply:
Solved!